src/nisarqa/products/product_reader.py

```diff
@@ -502,6 +502,42 @@ def product_spec_version(self) -> str:
 
             return spec_version
 
+    @cached_property
+    def runconfig_contents(self) -> str:
+        """
+        Contents (verbatim) of input granule's `runConfigurationContents`.
+
+        Returns the Dataset contents as-is, with no further processing.
+        It is a known issue that the L1/L2 product types use different formats
+        for `runConfigurationContents` (e.g. JSON, YAML). If that format is
+        updated within ISCE3, then this function will simply continue to
+        copy the Dataset's contents as-is.
+
+        Returns
+        -------
+        runconfig_contents : str
+            Contents (verbatim) of input granule's `runConfigurationContents`.
+            If the product does not contain that Dataset (such as for older
+            datasets), "N/A" is returned.
+        """
+
+        path = Path(self._processing_info_metadata_group_path) / "parameters"
+        path = str(path)
+        with h5py.File(self.filepath) as f:
+            if "runConfigurationContents" in f[path]:
+                runconfig = f[path]["runConfigurationContents"][...]
+                runconfig = nisarqa.byte_string_to_python_str(runconfig)
+                return runconfig
```

Comment:
How about one less line of code?

```suggestion
                return nisarqa.byte_string_to_python_str(runconfig)
```